### DISCUSSION TITLE
Set $logName attribute dynamically?

### DISCUSSION THREAD

**Discussion by RezDev94** [spatie/laravel-activitylog#741] [Q&A] [answered]
Can I set the `$logName` dynamically?

**Comment by Gummibeer** [comment, marked as answer]
Hey,

I assume that you refer to the model events? If so you have multiple ways.
* override `getLogNameToUse()` method on model and define your logic
* use `tapActivity()` method on model and define your logic

These two will work out of the box and without major conflicts in future updates.
